app/src/main/res/layout/fragment_review_contest.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_margin="@dimen/activity_vertical_margin"
+        android:orientation="vertical"
+        >
+
+        <TextView
+            android:id="@+id/contest_title_hint"
+            style="@style/AppTheme.TextViewLabel"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/name"
+            android:textColor="@color/colorLightGray"
+            />
+
+        <TextView
+            android:id="@+id/contest_title_button"
+            style="@style/AppTheme.TextViewLabel"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:clickable="true"
+            />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="@color/colorLightGray"
+            />
+
+        <TextView
+            android:id="@+id/contest_description_hint"
+            style="@style/AppTheme.TextViewLabel"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/description"
+            android:textColor="@color/colorLightGray"
+            />
+
+        <TextView
+            android:id="@+id/contest_description_button"
+            style="@style/AppTheme.TextViewLabel"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:clickable="true"
+            />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="@color/colorLightGray"
+            />
+
+        <TextView
+            android:id="@+id/contest_categories_hint"
+            style="@style/AppTheme.TextViewLabel"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="0dp"
+            android:text="@string/categories"
+            android:textColor="@color/colorLightGray"
+            tools:visibility="visible"
+            />
+
+        <include layout="@layout/include_recycler_view"/>
```

Comment:
Actually can you change this layout title for something specific to the review screen?